### PR TITLE
Update Nations.json

### PR DESCRIPTION
**Pull request by Smashfanful**
Changed some lines, and also eliminated that mod Swedish civilization

### CODE REVIEW

File: android/assets/jsons/Civ V - Vanilla/Nations.json [hunk 1]
```diff
@@ -306,7 +306,7 @@
 		"startIntroPart2": "Gandhi, your people look to you to lead them to even greater heights of glory! Can you help your people realize their great potential, to once again become the world's center of arts, culture and religion? Can you build a civilization that will stand the test of time?",
 		"declaringWar": "I have just received a report that large numbers of my troops have crossed your borders.",
 		"attacked": "My attempts to avoid violence have failed. An eye for an eye only makes the world blind.",
-		"defeated": "You can chain me, you can torture me, you can even destroy this body, but you will never imprison my mind. ",
```
Comment (yairm210) [inline android/assets/jsons/Civ V - Vanilla/Nations.json:309]: Why change this?

File: android/assets/jsons/Civ V - Vanilla/Nations.json [hunk 2]
```diff
@@ -356,9 +356,9 @@
...
 		"startIntroPart2": "Mighty Sultan, heed the call of your people! Bring your empire back to the height of its power and glory and once again the world will look upon your greatness with awe and admiration! Will you accept the challenge, great emperor? Will you build an empire that will stand the test of time?",
-		"declaringWar": "Your continued insolence and failure to recognize and preeminence leads us to war.",
```
Comment (yairm210) [inline android/assets/jsons/Civ V - Vanilla/Nations.json:359]: These too, why change? This works fine

Comment (Smashfanful) [reply android/assets/jsons/Civ V - Vanilla/Nations.json:359]: It was more faithful to the original Civ V. I'm soory if I bothered you with both these changes.

**Comment by yairm210** [comment]
I think the current lines are fine, and changing them would mean having to retranslate them in all languages.
Cons > pros.

**Pull request closed without merging**